tests/bigint.hpp

```diff
@@ -0,0 +1,146 @@
+/**
```

Comment:
there is no need to write this file again and again you can just include it direclty in your code like
```
#include <bigint.hpp>
```
please see the sample_test example in `test/operations/`